core-web/apps/dotcms-ui/src/app/portlets/dot-edit-page/layout/dot-edit-layout/dot-edit-layout.component.ts

```diff
@@ -119,11 +120,25 @@ export class DotEditLayoutComponent implements OnInit, OnDestroy {
      * @param {DotLayout} value
      * @memberof DotEditLayoutComponent
      */
-    nextUpdateTemplate(value: DotLayout) {
+    nextUpdateTemplate(value: DotTemplateDesignerPayload) {
         this.canRouteBeDesativated(false);
         this.updateTemplate.next(value);
     }
 
+    /**
+     * Update layout
+     *
+     * @param {DotLayout} layout
+     * @memberof DotEditLayoutComponent
+     */
+    onLayoutChange(layout: DotLayout) {
+        this.nextUpdateTemplate({
```

Comment:
100% sure that this code will not affect the current? should be hide the feature flag somehow?

Reply:
This is not going to affect the current Template Build. This function takes a `layout` and updates the template. 

The function was created because unlike the old template builder, which emits the entire `template`, the new one only emits the `layout`.